inc/saiwred.h

```diff
@@ -1,300 +1,169 @@
-/**
- * Copyright (c) 2014 Microsoft Open Technologies, Inc.
- *
- *    Licensed under the Apache License, Version 2.0 (the "License"); you may
- *    not use this file except in compliance with the License. You may obtain
- *    a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
- *
- *    THIS CODE IS PROVIDED ON AN *AS IS* BASIS, WITHOUT WARRANTIES OR
- *    CONDITIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT
- *    LIMITATION ANY IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS
- *    FOR A PARTICULAR PURPOSE, MERCHANTABLITY OR NON-INFRINGEMENT.
- *
- *    See the Apache Version 2.0 License for specific language governing
- *    permissions and limitations under the License.
- *
- *    Microsoft would like to thank the following companies for their review and
- *    assistance with these files: Intel Corporation, Mellanox Technologies Ltd,
- *    Dell Products, L.P., Facebook, Inc
- *
- * @file    saiwred.h
- *
- * @brief   This module defines SAI QOS Wred interface
- */
+/*
+* Copyright (c) 2015 Dell Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may
+* not use this file except in compliance with the License. You may obtain
+* a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+*
+* THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR
+* CONDITIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT
+* LIMITATION ANY IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS
+* FOR A PARTICULAR PURPOSE, MERCHANTABLITY OR NON-INFRINGEMENT.
+*
+* See the Apache Version 2.0 License for specific language governing
+* permissions and limitations under the License.
+*
+* @file saiwred.h
+*
+* @brief This file contains Qos Wred functionality.
+************************************************************************/
+
```

Comment:
I'm not signing off on this review, as removing the Microsoft copyright seems very wrong to me.  Please confirm with legal types.